automatic/qbittorrent/tools/chocolateyinstall.ps1

```diff
@@ -2,25 +2,20 @@
 
 $toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
-$fileName = if ((Get-OSArchitectureWidth 64) -and ($env:chocolateyForceX86 -ne 'true')) {
-  Write-Host "Installing 64 bit version" ; 'qbittorrent_4.1.1_x64_setup.exe' # 64-bit
-} else {
-  Write-Host "Installing 32 bit version" ; 'qbittorrent_4.1.1_setup.exe' # 32-bit
-}
-
 $packageArgs = @{
   packageName    = 'qbittorrent'
   fileType       = 'exe'
   softwareName   = 'qBittorrent*'
-  file           = "$toolsDir\$fileName"
+  file           = "$toolsDir\qbittorrent_4.1.1_setup.exe"
+  file64         = "$toolsDir\qbittorrent_4.1.1_x64_setup.exe"
   silentArgs     = '/S'
   validExitCodes = @(0, 1223)
 }
 
 Install-ChocolateyInstallPackage @packageArgs
 
 # Lets remove the installer as there is no more need for it
-Remove-Item -Force "$toolsDir\*.exe","$toolsDir\*.exe.ignore" -ea 0
+Get-ChildItem $toolsDir\*.exe | ForEach-Object { Remove-Item $_ -ea 0; if (Test-Path $_) { Set-Content "$_.ignore" '' } }
```

Comment:
Just a tip, you don't need to use the full name of the cmdlets.
Aliases is automatically expanded when a package is updated.

Reply:
Sure, but I find it more easier to read the code when the cmdlets are expanded. A personal matter of choice :)

Reply:
fair enough.
Just wanted to let you know just in case.

Reply:
also, just a little self promotion.
If you come across some scripts you want to expand aliases, and don't want to do it by hand.
Then you can download my powershell script through [`choco install wormies-au-helpers`](https://chocolatey.org/packages/wormies-au-helpers) (not really dependent on AU), where there is a helper script called [`Expand-Aliases`](https://wormiecorp.github.io/Wormies-AU-Helpers/docs/functions/expand-aliases) that you can use. (allows you to expand scripts in a directory, distinct files or just plain text).